src/sql/src/session/vars/definitions.rs

```diff
@@ -432,6 +432,13 @@ pub static MAX_POSTGRES_CONNECTIONS: VarDefinition = VarDefinition::new(
     false,
 );
 
+pub static MAX_MYSQL_CONNECTIONS: VarDefinition = VarDefinition::new(
+    "max_mysql_connections",
+    value!(u32; 0),
```

Comment:
Your life will probably be much easier if you default this to 1000, and then use LD in production to set it back to 0 by default for production environments.

Reply:
ah great idea - will do

Reply:
sounds good, set up the LD flag in prod to do so